manifests/b/bencoepp/kommod/1.0.0.0/bencoepp.kommod.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: bencoepp.kommod
+PackageVersion: 1.0.0.0

```

```suggestion
PackageVersion: 0.0.0.1
```
This should match what the package reports as it's own version
```
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> kommod -v
0.0.0.1
```